packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -397,3 +395,8 @@ new-user-submit = Subscribe Now
 
 manage-pocket-title = Looking for your { -brand-name-pocket } premium subscription?
 manage-pocket-body = To manage it, <a>click here</a>.
+
+payment-method-header = Choose your payment method
+## $prefix (string) - If header is part of a multi step process and needs a header. eg. '2.'
```

Comment:
I think it should be just the one `#` here.  (See https://projectfluent.org/fluent/guide/comments.html)

Reply:
Ah cool. Thank you for sharing the link. I had missed that.